enterprise/backend/test/metabase_enterprise/content_management/api/collection_test.clj

```diff
@@ -0,0 +1,100 @@
+(ns metabase-enterprise.content-management.api.collection-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.models.collection :as collection]
+   [metabase.public-settings.premium-features-test :as premium-features-test]
+   [metabase.test :as mt]
+   [toucan2.core :as t2]
+   [toucan2.tools.with-temp :as t2.with-temp]))
+
+(deftest create-collection-authority-test
+  (testing "POST /api/collection/:id"
+    (testing "when :content-management is enabled "
+      (premium-features-test/with-premium-features #{:content-management}
+        (testing "Admins add an official collection"
+          (mt/with-model-cleanup [:model/Collection]
+            (let [resp (mt/user-http-request :crowberto :post 200 "collection" {:name            "An official collection"
+                                                                                :color           "#000000"
+                                                                                :authority_level "official"})]
+              (is (= :official (t2/select-one-fn :authority_level :model/Collection (:id resp))))))
+
+          (testing "but the type has to be valid"
+            (mt/user-http-request :crowberto :post 400 "collection"
+                                  {:name "foo" :color "#f38630",:authority_level "invalid-type"})))
+
+        (testing "non-admins get 403"
+          (is (= "You don't have permissions to do that."
+                 (mt/user-http-request :rasta :post 403 "collection" {:name            "An official collection"
+                                                                      :color           "#000000"
+                                                                      :authority_level "official"}))))))
+
+    (testing "fails to add an official collection if doesn't have any premium features"
+      (premium-features-test/with-premium-features #{}
+        (is (= "Official collection is an Enterprise feature. Please upgrade to a paid plan to use this feature."
+               (mt/user-http-request :crowberto :post 402 "collection" {:name            "An official collection"
+                                                                        :color           "#000000"
+                                                                        :authority_level "official"})))))))
+
+(deftest update-collection-authority-happy-path-test
+  (testing "PUT /api/collection/:id"
+    (testing "update authority_level when has :content-management feature"
+      (premium-features-test/with-premium-features #{:content-management}
+        (testing "requires admin"
+          (t2.with-temp/with-temp
+            [:model/Collection {id :id} {:authority_level nil}]
```

Comment:
yea that looks better if there is one item. but if you have multiple items this list gets rather long because we need to align 3 columns.

I prefer to use that convention everywhere in case we need to expand the with-temp, then we don't have to change much of the structure.